### PR TITLE
Added more info about volumes

### LINKED ISSUE #344: Improve/expand info on volumes, `docker volume` and related concepts
URL: [https://docs.docker.com/engine/tutorials/dockervolumes.md](https://docs.docker.com/engine/tutorials/dockervolumes/)

Can someone expand on `docker volume` and write a separate section on how to manage those since that's the new preferred method of volume management?  I would contribute some edits myself but I'm only getting to know it now.  

Specifically information regarding **backup/restore** and **backups/restores** with **labels** and things like that.

I've managed to sort out this information on my own but someone coming to this guide for the first time might get confused with the focus mainly being on the `docker create -v` and `docker create --name data busybox` and `docker volume create` all of these methods are great and important to keep documented but ideally we should have something set to be the primary / recommended vs deprecated methods.

### PR DESCRIPTION
**Pull request by surajnarwade**
Resolves #344

https://docs.docker.com/engine/tutorials/dockervolumes/

### CODE REVIEW

File: engine/tutorials/dockervolumes.md [hunk 1]
```diff
@@ -332,6 +332,19 @@ $ docker run --rm --volumes-from dbstore2 -v $(pwd):/backup ubuntu bash -c "cd /
 You can use the techniques above to automate backup, migration and
 restore testing using your preferred tools.
 
+## Listing all volumes
```
Comment (mstanleyjones) [inline engine/tutorials/dockervolumes.md]: We try not to use the gerund format (-ing) in titles, so please change this to "List all volumes"

File: engine/tutorials/dockervolumes.md [hunk 2]
```diff
@@ -332,6 +332,19 @@ $ docker run --rm --volumes-from dbstore2 -v $(pwd):/backup ubuntu bash -c "cd /
...
+
+You can list out all volumes which are created.
```
Comment (mstanleyjones) [inline engine/tutorials/dockervolumes.md]: Maybe "List all existing volumes:"

File: engine/tutorials/dockervolumes.md [hunk 3]
```diff
@@ -347,6 +360,14 @@ $ docker run --rm -v /foo -v awesome:/bar busybox top
 This command creates an anonymous `/foo` volume. When the container is removed,
 the Docker Engine removes the `/foo` volume but not the `awesome` volume.
 
+If you want to remove all unused volumes, which are unnecessarily occupying the space,
```
Comment (mstanleyjones) [inline engine/tutorials/dockervolumes.md]: Maybe something like "To remove unused volumes and free up space:"

**Comment by londoncalling** [comment]
LGTM, it looks like @mstanleyjones  comments are addressed, merging

**Pull request merged**